docs/getting_started.rst

```diff
@@ -88,6 +88,28 @@ following statements hold true:
                      connect=True,
                      auto_renew=True)
 
+
+Identity and Access Management (IAM)
+####################################
+
+IBM Cloud Identity & Access Management enables you to securely authenticate
+users and control access to all cloud resources consistently in the IBM Bluemix
+Cloud Platform.
+
+See `Managing Identity and Access <https://console.bluemix.net/docs/iam/users_roles.html#userroles>`_
+for more information.
+
+You can easily connect to your Cloudant account using an IAM API key:
+
+.. code-block:: python
+
+    # Authenticate using an IAM API key
+    client = Cloudant(USERNAME, API_KEY, account=ACCOUNT_NAME,
+                     auto_renew=True,
+                     connect=True,
+                     use_iam=True)
```

Comment:
I'm not a fan of exposing yet another parameter on here, I think it is particularly confusing that the `USERNAME` is effectively ignored in this scenario. I wonder whether we can have another way of initializing that effectively hides this complexity.